src/Shared/Recruit.Vacancies.Client/Infrastructure/QueryStore/QueryStoreClient.cs

```diff
@@ -96,17 +97,66 @@ public Task<EditVacancyInfo> GetEmployerVacancyDataAsync(string employerAccountI
 
         public Task<ProviderEditVacancyInfo> GetProviderVacancyDataAsync(long ukprn)
         {
+            //TODO get data from QueryStore Collection
```

Comment:
💥 We shouldn't need TODO's, the code/signatures and execution will tell us what is missing.